server/src/main/java/org/opensearch/cluster/metadata/MetadataCreateIndexService.java

```diff
@@ -250,6 +250,10 @@ public static void validateIndexOrAliasName(String index, BiFunction<String, Str
         if (!Strings.validFileName(index)) {
             throw exceptionCtor.apply(index, "must not contain the following characters " + Strings.INVALID_FILENAME_CHARS);
         }
+        if (index.isEmpty()) {
+            logger.info("Invalid Index Name [], must not be empty");
```

Comment:
Should probably omit the log statement. The other cases in this method do not log here. A misconfigured client could cause your log to grow to a huge size if you log every invalid request.